CHANGELOG.yml

```
@@ -6,6 +6,7 @@ upcoming:
   user_facing:
     - Adds <ArtistConsignButton> to Artist page - chris
     - Show verification text and FAQ link in auctions - brian
+    - Non-IDV new bidder registering through typical registration form sees information about identity verification - yuki
```

```suggestion
    - Non-IDV bidder registering through typical registration form sees information about identity verification - yuki
```

Suggested removing the "new" in "new bidder" – wasn't sure what you meant by that? For returning (non-new) bidders, that already have a CC, we're still changing the experience it seems.